### ISSUE TITLE
a11y: [Epic]: Aria Labels, label associations, roles and alt tags

### ISSUE THREAD

**Issue by kwwii** [rancher/dashboard#7968]
All non-functional images should have alt tags to describe them (or remove useless images or mark them as decorative for screen readers). Add Aria labels for all icons, icon buttons, sorting arrows and menu chevron/carets.

- [x] https://github.com/rancher/dashboard/issues/8113
- [x] https://github.com/rancher/dashboard/issues/12804
- [x] https://github.com/rancher/dashboard/issues/12805
- [x] https://github.com/rancher/dashboard/issues/12806
- [x] https://github.com/rancher/dashboard/issues/12807
- [x] https://github.com/rancher/dashboard/issues/12810
- [x] https://github.com/rancher/dashboard/issues/12812
- [x] https://github.com/rancher/dashboard/issues/12816
- [x] https://github.com/rancher/dashboard/issues/12817
- [x] https://github.com/rancher/dashboard/issues/12818

Out of scope for 2.11:

- [ ] https://github.com/rancher/dashboard/issues/8114
- [ ] https://github.com/rancher/dashboard/issues/8146
- [ ] https://github.com/rancher/dashboard/issues/8147
- [ ] https://github.com/rancher/dashboard/issues/8160
- [ ] https://github.com/rancher/dashboard/issues/8191

**Comment by kwwii** [comment]
https://github.com/rancher/dashboard/issues/4378
https://github.com/rancher/dashboard/issues/4387

**Comment by rancher-max** [comment]
@gaktive Looks like the v2.11 tickets have been closed, so we can probably update the milestone here to v2.12 or another future milestone to cover the other issues.

**Comment by rak-phillip** [comment]
Closing this epic. All of the 2.11 items have been addressed. Additional work has been added to the 2.12 milestone.
